tick/solver/saga.py

```diff
@@ -152,10 +152,10 @@ class SAGA(SolverFirstOrderSto):
     time_end : `str`
         End date of the call to ``solve()``
 
-    dtype : `string`, default='float64'
-        Type of arrays to use - default float64
+    dtype : `{'float64', 'float32'}`, default='float64'
+        Type of the arrays used. This value is set from model and prox dtypes.
 
-    var_red_str : `string`
+    _var_red_str : `string`
```

Comment:
We rather put private variables documentation near to their definition (just before line 178)